mediathread/assetmgr/features/collection.feature

```diff
@@ -35,8 +35,8 @@ Feature: Collection View
         And the Collection panel has a "Mediathread: Introduction" item
 
         When I clear all tags
-        When I select "Student One" as the owner
-        Then the owner is "Student One" in the Collection column
+        When I select "One, Student" as the owner
```

Comment:
Oi! I didn't think about these tests when we talked about changing the display.

Reply:
Yeah, changing to last name, first name for name display will make some things feel less natural. This test stuff doesn't matter too much though.. "Student One", "Instructor One", etc, are only used in testing.